core/common/assets/js/utils/debug.js

```diff
@@ -19,9 +19,12 @@ var Debug = function() {
 	};
 
 	var onError = function( event ) {
-		var originalEvent = event.originalEvent,
-			error = originalEvent.error;
+		var originalEvent = event.originalEvent;
+		if ( ! originalEvent ) {
+			return;
+		}
 
+		var error = originalEvent.error;
```

Comment:
You can combine it to a single line:

```suggestion
		const error = event.originalEvent?.error;
```

Reply:
Of course, just had a feeling you didnt support optional chaining. 😅